webpack.config.js

```diff
@@ -46,29 +50,43 @@ if (LIFECYCLE_EVENT == 'test' || LIFECYCLE_EVENT == 'test:watch') {
   process.env.BABEL_ENV = 'test';
 }
 
-const CSS_LOADER_CONFIG = [
+const SASS_LOADER_CONFIG = [
   {
     loader: 'css-loader',
     options: {
-      sourceMap: true,
+      sourceMap: GENERATE_SOURCE_MAPS,
     },
   },
   {
     loader: 'postcss-loader',
     options: {
-      sourceMap: IS_DEV,
+      sourceMap: GENERATE_SOURCE_MAPS,
       plugins: () =>[require('autoprefixer')({grid: false})],
     },
   },
   {
     loader: 'sass-loader',
     options: {
-      sourceMap: true,
+      sourceMap: GENERATE_SOURCE_MAPS,
       includePaths: glob.sync('packages/*/node_modules').map((d) => path.join(__dirname, d)),
     },
   },
 ];
 
+const CSS_WRAPPED_IN_JS_CONFIG = [{loader: 'style-loader'}].concat(SASS_LOADER_CONFIG);
```

Comment:
How come this was made into a constant when it's only used in one place, and the other side of the ternary where this is used doesn't also have a constant?  (Personally I'd vote for neither being a constant since it's just one place, but both being constants could also be valid.)